osu.Game/Screens/Select/SongSelect.cs

```diff
@@ -649,6 +675,8 @@ public override void OnSuspending(ScreenTransitionEvent e)
 
         public override bool OnExiting(ScreenExitEvent e)
         {
+            backgroundBlurLevel.ValueChanged -= applyBackgroundBlur;
```

Comment:
same as above, i dont think you are understanding how bindables work unless im misunderstanding whats being attempted here. Valuechange is automatically called whenever anything happens in code that alters the bindables value, its not something you use to inform the bindable that you want to change the value. so im not sure why its being done here, when resuming and when exiting. if you want to set the value of the bindable when these things happen then it should be  bindablename.value  = whatevervalue;

Reply:
By that I'm unsubscribing to the event, preventing this setting to affect background blur of the playfield for example when changing the setting value while playing. Since (from my understanding) the `BackgroundScreenBeatmap` is shared in a lot of contexts, this is the cleanest way I could think of to prevent unwanted blur.
Basically, I wanted to be subscribed to the `ValueChanged` event only when it was necessary.

Reply:
right but this is all being set on song selects background, ill test now but i dont think itd propagate once you left it

Reply:
it does unfortunately...